app/components/cases/new-subcase/component.js

```diff
@@ -75,11 +75,11 @@ export default Component.extend({
   },
 
   createSubcaseObject(newCase, newDate) {
-    const { type, title, shortTitle, confidential, showAsRemark } = this;
+    let { type, title, shortTitle, confidential, showAsRemark } = this;
```

Comment:
Can be `const`

Reply:
I just checked, and for some reason our eslint does not enforce this. So you can resolve this as far as I'm concerned!

Reply:
Ok, weird that the linter doesn't enforce this.. will need to look into that one for sure!